site.mk

```diff
@@ -1,12 +1,11 @@
 GLUON_FEATURES := \
 	autoupdater \
+	config-mode-outdoor \
 	ebtables-filter-multicast \
 	ebtables-filter-ra-dhcp \
-	ebtables-limit-arp \
 	ebtables-source-filter \
 	mesh-batman-adv-14 \
 	mesh-vpn-fastd \
-	radvd \
```

Comment:
What does it do in current releases? Not sure if we don't break anything...

Reply:
This is now included by default so not needed anymore here